src/stylesheets/components/_mobile-navigation.scss

```diff
@@ -0,0 +1,129 @@
+.app-c-mobile-nav {
+  display: none;
+
+  &--active {
+    display: block;
+
+    @include mq($from: desktop) {
+      display: none;
+    }
+  }
+
+  &.app-c-mobile-nav--hide-on-pageload {
+    display: none; // Prevent flash of content, this gets removed with JS
+  }
+
+  .no-js & {
+    @include mq($until: desktop) {
+      display: block;
+    }
+  }
+}
+
+// No JS fallback to aid with navigation
+.app-c-mobile-nav__breadcrumbs {
+  display: none;
+  padding: 16px $govuk-spacing-scale-4 17px $govuk-spacing-scale-4;
+
+  .no-js & {
+    display: block;
+  }
+}
+
+.app-c-mobile-nav__list {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  background-color: $app-light-grey;
+
+
+  & > li {
+    @include govuk-bold-19;
+    @include font-smoothing;
+    font-family: $govuk-font-stack;
+    font-size: 19px; // We don't have a font mixin that produces 19px on mobile
```

Comment:
@dashouse Are there things we should be doing with the navigation to make it fit with the conventions we have in Frontend? There's quite a lot of magic numbers in here.

Reply:
Just chatted to you @36degrees. @dashouse can revisit this later.